Can not make lshort.pdf

### Environment:

ArchLinux with:

```sh
$ pacman -Qs texlive
local/texlive-bibtexextra 2023.66579-1 (texlive-most)
    TeX Live - Additional BibTeX styles and bibliography databases
local/texlive-bin 2023.66539-1
    TeX Live binaries
local/texlive-core 2023.66587-2 (texlive-most)
    TeX Live core distribution
local/texlive-fontsextra 2023.66328-1 (texlive-most)
    TeX Live - all sorts of extra fonts
local/texlive-formatsextra 2023.66186-1 (texlive-most)
    TeX Live - collection of extra TeX 'formats'
local/texlive-games 2023.66190-1 (texlive-most)
    TeX Live - Setups for typesetting various board games, including chess
local/texlive-humanities 2023.65502-1 (texlive-most)
    TeX Live - LaTeX packages for law, linguistics, social sciences, and
    humanities
local/texlive-langchinese 2023.66188-1 (texlive-lang)
    TeX Live - Fonts and macro packages to typeset Chinese texts
local/texlive-langextra 2023.66225-1 (texlive-lang)
    TeX Live - Bundle of all extra language support
local/texlive-langjapanese 2023.66482-1 (texlive-lang)
    TeX Live - Fonts and macro packages to typeset Japanese texts
local/texlive-latexextra 2023.66551-1 (texlive-most)
    TeX Live - Large collection of add-on packages for LaTeX
local/texlive-music 2023.66278-1 (texlive-most)
    TeX Live - Music typesetting packages
local/texlive-pictures 2023.66549-1 (texlive-most)
    TeX Live - Packages for drawings graphics
local/texlive-pstricks 2023.66115-1 (texlive-most)
    TeX Live - Additional PSTricks packages
local/texlive-publishers 2023.66550-1 (texlive-most)
    TeX Live - LaTeX classes and packages for specific publishers
local/texlive-science 2023.66461-1 (texlive-most)
    TeX Live - Typesetting for mathematics, natural and computer sciences

$ pacman -Q make
make 4.4.1-2
$ pacman -Q python-pygments
python-pygments 2.14.0-2
$ pacman -Q git
git 2.40.0-1
```

### Symptom:

Can't finish `make lshort.pdf` after clean git clone and cd.

### Error output:

(... just the end of it)

```
warning  (pdf backend): ignoring duplicate destination with the name 'page.2'
[2]
warning  (pdf backend): ignoring duplicate destination with the name 'page.3'
[3]

LaTeX Warning: Citation 'companion' on page 4 undefined on input line 204.


warning  (pdf backend): ignoring duplicate destination with the name 'page.4'
[4]

LaTeX Warning: Reference `installinglatex' on page 5 undefined on input line 24
0.

(./pdfbuild/minted_cache/default.pygstyle)Error: no lexer for alias '\\__lshortexample_minted_language ' found


! Package minted Error: Missing Pygments output; \inputminted was
probably given a file that does not exist--otherwise, you may need 
the outputdir package option, or may be using an incompatible build tool,
or may be using frozencache with a missing file.

See the minted package documentation for explanation.
Type  H <return>  for immediate help.
 ...                                              
                                                  
l.262 \end{example}
                 
 19479 words of node memory still in use:
   93 hlist, 11 vlist, 4 rule, 1 ins, 3 mark, 109 disc, 15 local_par, 2 dir, 4 
math, 477 glue, 118 kern, 62 penalty, 22 margin_kern, 1605 glyph, 1 page_insert
, 222 attribute, 120 glue_spec, 222 attribute_list, 2 temp, 3 if_stack, 1 shape
, 8 write, 1 pdf_start_link, 1 pdf_end_link, 3 pdf_dest, 3 pdf_action, 18 pdf_c
olorstack nodes
   avail lists: 1:2,2:284,3:303,4:80,5:433,6:28,7:4457,8:50,9:348,10:11,11:285
!  ==> Fatal error occurred, no output PDF file produced!
Transcript written on lshort.log.
make: *** [Makefile:35: lshort.pdf] Error 1
```

--- a/.chktexrc
+++ b/.chktexrc
@@ -1,5 +1,5 @@
 CmdLine {
-    -eall -n22 -n30 -e16 -n46 -n21 -n29 -n3 -n17 -n11
+    -eall -n22 -n30 -e16 -n46 -n21 -n29 -n3 -n17 -n11 -n12
 }
 
 VerbEnvir {
@@ -13,6 +13,7 @@ Silent {
     \tabularnewline \ifpdf \ExplSyntaxOn \ExplSyntaxOff
     \bgroup \egroup \sourcefamily \restoregeometry
     \itshape \cleardoublepage \IfValueT \pause \blindtext
+    \backmatter
 }
 
 MathEnvir {

--- a/src/bibliography.tex
+++ b/src/bibliography.tex
@@ -395,10 +395,10 @@ options include
 \end{example}
 
 As you may have noticed, when entries have many authors, then not all of
-them will be printed; instead ``et al.'' shows up at the end of the author list. This behaviour
+them will be printed; instead \enquote{et al.} shows up at the end of the author list. This behaviour
 may be controlled via \cargv{maxnames} (default $3$) and \cargv{minnames}
 (default $1$) options. If the number of names is greater than \carg{maxnames}
-then it will be shortened to \carg{minnames} and ``et al.'' will be added.
+then it will be shortened to \carg{minnames} and \enquote{et al.} will be added.
 
 \begin{example}[standalone,
   biber,

--- a/src/deprecated.tex
+++ b/src/deprecated.tex
@@ -55,8 +55,8 @@
   \begin{trivlist}
     \item
     % Workaround for https://github.com/gpoore/minted/issues/356
-    \RenewCommandCopy\VerbEnv\Verbatim
-    \RenewCommandCopy\endVerbEnv\endVerbatim
+    \RenewCommandCopy{\VerbEnv}{\Verbatim}
+    \RenewCommandCopy{\endVerbEnv}{\endVerbatim}
     \begin{minipage}{0.4\textwidth}
       \mintinline{latex}:#1:
     \end{minipage}
@@ -71,8 +71,8 @@
 \NewDocumentCommand{\vchto}{+v+v}{
   \begin{center}
     % Workaround for https://github.com/gpoore/minted/issues/356
-    \RenewCommandCopy\VerbEnv\Verbatim
-    \RenewCommandCopy\endVerbEnv\endVerbatim
+    \RenewCommandCopy{\VerbEnv}{\Verbatim}
+    \RenewCommandCopy{\endVerbEnv}{\endVerbatim}
     \mintinline{latex}:#1:
     \(\big\downarrow\)\\[.2cm]
     \mintinline{latex}:#2:

--- a/src/deprecated.tex
+++ b/src/deprecated.tex
@@ -54,6 +54,9 @@
 \NewDocumentCommand{\chto}{+v+v}{
   \begin{trivlist}
     \item
+    % Workaround for https://github.com/gpoore/minted/issues/356
+    \RenewCommandCopy\VerbEnv\Verbatim
+    \RenewCommandCopy\endVerbEnv\endVerbatim
     \begin{minipage}{0.4\textwidth}
       \mintinline{latex}:#1:
     \end{minipage}
@@ -67,7 +70,10 @@
 }
 \NewDocumentCommand{\vchto}{+v+v}{
   \begin{center}
-    \mintinline{latex}:#1:\\[.2cm]
+    % Workaround for https://github.com/gpoore/minted/issues/356
+    \RenewCommandCopy\VerbEnv\Verbatim
+    \RenewCommandCopy\endVerbEnv\endVerbatim
+    \mintinline{latex}:#1:
     \(\big\downarrow\)\\[.2cm]
     \mintinline{latex}:#2:
   \end{center}
@@ -155,7 +161,7 @@ optional argument with default value (specified as second optional argument)
 followed by few mandatory ones (specified as number in first optional
 argument).
 \begin{chktexignore}
-  \vchto|\newcommand{\foo}[4][bar]{ ... }|
+  \vchto|\newcommand{\foo}[4][bar]{ ... }|%
   |\NewDocumentCommand{\foo}{O{bar}mmmm}{ ... }|
 \end{chktexignore}
 

--- a/src/lshort.sty
+++ b/src/lshort.sty
@@ -63,6 +63,7 @@
   Script=Arabic,
 }
 \newfontfamily\arabicfonttt{Iran Nastaliq}
+\newfontfamily\arabicfont{Iran Nastaliq}
 \newfontface\@swash[
   Style=Swash,
 ]{EB Garamond}

--- a/src/lshortexample.sty
+++ b/src/lshortexample.sty
@@ -368,3 +368,10 @@
     }
   }
 }
+
+% Workaround for https://github.com/gpoore/minted/issues/354
+\ifwindows
+  \def\minted@opt@quote#1{\detokenize\expandafter{\expandafter"\expanded{#1}"}}
+\else
+  \def\minted@opt@quote#1{\detokenize\expandafter{\expandafter'\expanded{#1}'}}
+\fi

--- a/src/realworld.tex
+++ b/src/realworld.tex
@@ -333,16 +333,16 @@ Sch\"onbrunner Schlo\ss, \.Z\'o\l\'c
   \caption{Accents and Special Characters.}\label{accents}
   \begin{tabular}{@{}*3{ll@{\qquad}}ll@{}}
     \toprule
-    Code         & Result       & Code          & Result       & Code & Result    & Code & Result \\
+    Code         & Result       & Code          & Result       & Code      & Result & Code & Result \\
     \midrule
-    \mstA{\`o}   & \mstA{\'o}   & \mstA{\^o}    & \mstA{\~o}                                      \\
-    \mstA{\=o}   & \mstA{\.o}   & \mstA{\"o}    & \mstB{\c}{c}                                    \\[6pt]
-    \mstB{\u}{o} & \mstB{\v}{o} & \mstB{\H}{o}  & \mstB{\k}{a}                                    \\
-    \mstB{\d}{o} & \mstB{\b}{o} & \mstB{\t}{oo} & \mstB{\r}{o}                                    \\[6pt]
-    \mstA{\oe}   & \mstA{\OE}   & \mstA{\ae}    & \mstA{\AE}                                      \\
-    \mstA{\aa}   & \mstA{\AA}   &               &              &      &                           \\[6pt]
-    \mstA{\o}    & \mstA{\O}    & \mstA{\l}     & \mstA{\L}                                       \\
-    \mstA{\i}    & \mstA{\j}    & \verb|!`|     & !`    & \verb|?`| &  ?`                         \\ %chktex 26
+    \mstA{\`o}   & \mstA{\'o}   & \mstA{\^o}    & \mstA{\~o}                                        \\
+    \mstA{\=o}   & \mstA{\.o}   & \mstA{\"o}    & \mstB{\c}{c}                                      \\[6pt]
+    \mstB{\u}{o} & \mstB{\v}{o} & \mstB{\H}{o}  & \mstB{\k}{a}                                      \\
+    \mstB{\d}{o} & \mstB{\b}{o} & \mstB{\t}{oo} & \mstB{\r}{o}                                      \\[6pt]
+    \mstA{\oe}   & \mstA{\OE}   & \mstA{\ae}    & \mstA{\AE}                                        \\
+    \mstA{\aa}   & \mstA{\AA}   &               &              &           &                        \\[6pt]
+    \mstA{\o}    & \mstA{\O}    & \mstA{\l}     & \mstA{\L}                                         \\
+    \mstA{\i}    & \mstA{\j}    & \verb|!`|     & !`           & \verb|?`| & ?`                     \\ %chktex 26
     \bottomrule
   \end{tabular}%
   \index{dotless \i{} and \j}\index{Scandinavian letters}%

